packages/react/src/auto/AutoForm.ts

```diff
@@ -38,24 +37,26 @@ export type AutoFormProps<
 /**
  * React hook for getting the validation schema for a list of fields
  */
-export const useValidationResolver = (metadata: ActionMetadata | undefined) => {
+export const useValidationResolver = (metadata: ActionMetadata | GlobalActionMetadata | undefined) => {
   return useMemo(() => {
     if (!metadata) return undefined;
-    return yupResolver(validationSchema(metadata.action.inputFields));
+    const action = "action" in metadata ? metadata.action : metadata;
```

Comment:
Perhaps helpers like 
```
isGlobalAction(metadata)
getActionFromMetadata(metadata)
```
could be added so that it is a bit more obvious if the metadata is related to a global/model action without needing to look through the action metadata types

Reply:
Good call!